### DISCUSSION TITLE
[M-Z] (20 Gennaio 2020) Esercizio 10

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#479] [Esami & Esoneri M-Z]
### Traccia

<img width="936" alt="image" src="https://github.com/user-attachments/assets/92fd8503-4af7-4af9-8d0b-7185933045a1">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200120.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200120.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/78cfd3c0-fa4f-46b4-bb42-78d0e22d0a7e)
